pkg/api/localnetv1/services.proto

```diff
@@ -82,6 +82,15 @@ message Service {
     repeated PortMapping Ports = 6;
 
     bool ExternalTrafficToLocal = 7;
+
+    string SessionAffinity = 10;
+    SessionAffinityConfig SessionAffinityConfig = 11;
+    IPSet LoadBalancerSourceRanges = 12;
+    IPSet LoadBalancerIngressIPs = 13;
+}
+
+message SessionAffinityConfig {
```

Comment:
rename to SessionAffinity and add `string Kind = 1` that will receive the k8s `SessionAffinity` value